pkgs/applications/video/mpv/scripts/smartskip.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  buildLua,
+}:
+
+buildLua {
+  pname = "mpv-smartskip";
+  version = "0-unstable-2023-11-25";
+
+  scriptPath = "scripts/SmartSkip.lua";
+  src = fetchFromGitHub {
+    owner = "Eisa01";
+    repo = "mpv-scripts";
+    rev = "48d68283cea47ff8e904decc9003b3abc3e2123e";
+    sha256 = "sha256-95CAKjBRELX2f7oWSHFWJnI0mikAoxhfUphe9k51Qf4=";
+  };
+  passthru.scriptName = "SmartSkip.lua";
```

Comment:
```suggestion
```

when you specify `scriptPath`, then `buildLua` derives `scriptName` from the last component of that path, so specifying both here isn't necessary.